docs/050-command-reference.md

```diff
@@ -149,18 +149,40 @@ Serendipity commands are commands that help resolve to a random block from your
 
 - `<%RANDOMBLOCKMENTION:Evergreen Notes%>`
 
-## RANDOMCHILDOF
+## RANDOMCHILDOFMENTION
```

Comment:
maybe mention that this was previously called `RANDOMCHILDOF`? In case someone is searching in the reference for what their existing command does?